hudi-spark-datasource/hudi-spark3.2plus-common/src/main/scala/org/apache/spark/sql/hudi/catalog/HoodieStagedTable.scala

```diff
@@ -29,7 +30,7 @@ import org.apache.spark.sql.types.StructType
 
 import java.net.URI
 import java.util
-import scala.collection.JavaConverters.{mapAsScalaMapConverter, setAsJavaSetConverter}
+import scala.jdk.CollectionConverters.{mapAsScalaMapConverter, setAsJavaSetConverter}
```

Comment:
was this intentional change? I see we widely use `scala.collection.JavaConverters.` ? If you prefer to replace all usages,can we file a tracking jira and do it separately.

Reply:
`JavaConverters` are not available in Scala 2.11